README.md

```diff
@@ -8,12 +8,14 @@ Before starting to use the provider, it can be helpful to read through the gener
 and get familiar with the concepts. 
 
 ## Adding the dependency
+<!---x-release-please-start-version-->
```

Comment:
Unrelated to the goal of this PR, but it's good to have release-please also updating the readme